curvefs/conf/metaserver.conf

```diff
@@ -213,24 +213,29 @@ storage.type=rocksdb
 storage.max_memory_quota_bytes=32212254720
 # metaserver max disk quota bytes (default: 2TB)
 storage.max_disk_quota_bytes=2199023255552
-# storage root dir, rocksdb's all SST and other files store under it
-storage.data_dir=./storage  # __CURVEADM_TEMPLATE__ ${prefix}/data/storage __CURVEADM_TEMPLATE__
 # whether need to compress the value for memory storage (default: False)
 storage.memory.compression=False
+# rocksdb block cache(LRU) capacity (default: 8GB)
+storage.rocksdb.block_cache_capacity=8589934592
+# rocksdb writer buffer manager capacity (default: 6GB)
+storage.rocksdb.write_buffer_manager_capacity=6442450944
+# Control wether write buffer manager cost block cache
+# If true, the total memory usage by rocksdb is limited by `block_cache_capacity`
+storage.rocksdb.WBM_cost_block_cache=true
+# Maximum number of concurrent background jobs (compactions and flushes)
+storage.rocksdb.max_background_jobs=16
 # rocksdb column family's write_buffer_size
-# for store inode which exclude its s3chunkinfo list (unit: bytes, default: 128MB)
-storage.rocksdb.unordered_write_buffer_size=134217728
+# for store inode which exclude its s3chunkinfo list (unit: bytes, default: 64MB)
+storage.rocksdb.unordered_write_buffer_size=67108864
 # rocksdb column family's max_write_buffer_number
-# for store inode which exclude its s3chunkinfo list (default: 5)
-storage.rocksdb.unordered_max_write_buffer_number=5
+# for store inode which exclude its s3chunkinfo list (default: 3)
+storage.rocksdb.unordered_max_write_buffer_number=3
```

Comment:
can we increase the buffer number? is `3` is  enough?

Reply:
fix